ui/jobs/components/index.ts

```diff
@@ -156,14 +157,8 @@ export class ComponentManager {
     this.ee.on('party', (party) => this.partyTracker.onPartyChanged({ party }));
 
     this.player.on('level', (level, prevLevel) => {
-      if (level - prevLevel) {
-        this.bars._updateFoodBuff({
-          inCombat: this.component?.inCombat ?? false,
-          foodBuffExpiresTimeMs: this.foodBuffExpiresTimeMs,
-          foodBuffTimer: this.foodBuffTimer,
-          contentType: this.contentType,
-        });
-      }
+      if (level - prevLevel)
```

Comment:
I know this code was here before, but I think it means:
```suggestion
      if (level !== prevLevel)
```